doc/playbook.rst

```diff
@@ -21,16 +21,18 @@ Containers
 ==========
 
 The service consists of two containers.
-The first is a base container with the Science Pipelines "stack" code and networking utilities.
-The second is a service container made from the base that has the Prompt Processing service code.
+The first is a **base container** with the Science Pipelines "stack" code and networking utilities.
+The second is a **service container** made from the base that has the Prompt Processing service code.
 All containers are managed by `GitHub Container Registry <https://github.com/orgs/lsst-dm/packages?repo_name=prompt_processing>`_ and are built using GitHub Actions.
 
 To build the base container:
 
 * If there are changes to the container, push them to a branch, then open a PR.
-  The container should be built automatically.
-* If there are no changes (typically because you want to use an updated Science Pipelines container), go to the repository's `Actions tab <https://github.com/lsst-dm/prompt_processing/actions/workflows/build-base.yml>`_ and select "Run workflow".
-  From the dropdown, select the branch whose container definition will be used, and the label of the Science Pipelines container.
+  This will build the container automatically.
+* If there are no changes (typically because you want to use an updated Science Pipelines container), go to the repository's `Actions tab <https://github.com/lsst-dm/prompt_processing/actions/workflows/build-base.yml>`_ and select "**Run workflow**". From the dropdown menu, select:
+
+  #. The branch whose container definition will be used
+  #. The label of the Science Pipelines container.
```

Comment:
I think you need to include an entry for the science pipelines container (possibly adding it on commit "Add directions to playbook..."? AFAICT that's the step you and Erin missed over the weekend.

Reply:
Huh. [Sadly I was mostly out that weekend so I'll check in with @erinleighh about the specifics]. Do you mean that this set of directions needs the step where the container label is taken from the appropriate container at this link: https://github.com/orgs/lsst-dm/packages?repo_name=prompt_processing ?

Reply:
No, the action is hardcoded to always build `prompt-base`. But the "Science Pipelines container" needs to have the name of the _input_ container being used to make the base -- `lsstsqre/centos`, the default, is the official weekly/daily build, while Quick-Stack is [`ghcr.io/lsst/quick-stack`](http://ghcr.io/lsst/quick-stack) (you mentioned it earlier, as the output of the build process).